packages/router/src/provide_router.ts

```diff
@@ -730,6 +730,24 @@ export type ViewTransitionsFeature = RouterFeature<RouterFeatureKind.ViewTransit
  * );
  * ```
  *
+ * The router bindgs information from any of the following sources:
```

Comment:
nit: typo 

```suggestion
 * The router bindings information from any of the following sources:
```

 Very much needed/valuable explanation, Andrew 🙏🏻